src/steps/account/account-create.ts

```diff
@@ -1,5 +1,5 @@
-import { BaseStep, Field, StepInterface } from '../../core/base-step';
-import { Step, RunStepResponse, FieldDefinition, StepDefinition } from '../../proto/cog_pb';
+import { BaseStep, Field, StepInterface, ExpectedRecord } from '../../core/base-step';
+import { Step, RunStepResponse, FieldDefinition, StepDefinition, RecordDefinition } from '../../proto/cog_pb';
 
 export class CreateAccount extends BaseStep implements StepInterface {
```

Comment:
- [ ] Looks like the `AccountAware` mixin didn't get the same field treatment that the other objects got, so the info returned is still quite limited.

<img width="801" alt="Screen Shot 2020-02-16 at 5 16 01 PM" src="https://user-images.githubusercontent.com/3496491/74617045-09785980-50e0-11ea-8810-780b9cea3b0c.png">